cargo_all

```diff
@@ -4,5 +4,6 @@
 (cd octavo && cargo +nightly $*)
 (cd openssl && cargo +nightly $*)
 (cd ring && cargo +nightly $*)
-(cd dagenix && cargo +nightly $*)
+#un-maintained(cd dagenix && cargo +nightly $*)
+(cd RustCrypto && cargo +nightly $*)
```

Comment:
Let's move all the dagenix/rust-crypto changes to a separate PR and squash those commits into one.